src/app/core/models/session.model.ts

```diff
@@ -11,17 +11,29 @@ export interface SessionData {
 }
 
 export const SESSION_DATA_UNAVAILABLE = 'Session Data Unavailable';
-export const SESSION_DATA_INCORRECTLY_DEFINED = 'Session Data is Incorrectly Defined';
+const SESSION_DATA_MISSING_CRUCIAL_INFO = 'Session Data is missing crucial components';
```

Comment:
Similarly, I think it'll be more user friendly if the error message is more specific (perhaps `Session Data does not define any open phases`)

Reply:
Hmm that could be confusing because the first one is missing the `openPhases` key and the second has no open phases. The crucial info here refers to the first case. Perhaps `Session Data does not define openPhases` ? But thats more of a dev friendly than user friendly error message  😅 What do you think?

```typescript
const sessionDataNoOpenPhasesKey = {
  bugReporting: 'bugReporting'
}
const sessionDataNoOpenPhases = {
  openPhases: [],
  bugReporting: 'bugReporting'
}
```

Reply:
Ok, let's go with `Session data does not define an 'openPhases' key`